pprofile.py

```diff
@@ -883,14 +962,37 @@ def runpath(path, argv, filename=None, threads=True, verbose=False):
     """
     _run(threads, verbose, 'runpath', filename, path, argv)
 
-_allsep = os.sep + (os.altsep or '')
+def _qCacheGringAdapt(path):
+    '''
+    qCacheGrind (windows build) needs at least one UNIX separator
+    in path to find the file. Adapt here even if this is probably
+    more of a qCacheGrind issue...
+    '''
+
+    if os.path.sep != "/":
+        path = lambda x, cascade=path: cascade(
+            '/'.join(x.split(os.path.sep))
+        )
+
+    return path
+
+def _path_resolver(relative_path=True):
+    '''
+    Returns a function that manipulates a path string to be relative if
+    `relative_path' is true, otherwise it returns the identity function.
```

Comment:
Is it really worth making into a separate function ? It only save a "if" over verbatim parameter value and creating an identity lambda.
If does allow setting a default behaviour (via the default parameter value), but I see it more as a trap (caller forgetting to pass the parameter, rather than exposing it itself) than a feature.